vicinae/src/extension/extension-navigation-controller.hpp

```diff
@@ -33,7 +33,14 @@ class ExtensionNavigationController : public QObject {
 
   std::vector<ExtensionViewWrapper *> views() const { return m_views; }
 
-  void handleViewPoped() {
+  void handleViewPoped(const BaseView *view) {
```

Comment:
it's fine for now but we will have to think about something else, because that's definitely not pretty